readthedocs/config/config.py

```diff
@@ -110,16 +110,24 @@ class InvalidConfig(ConfigError):
     message_template = 'Invalid "{key}": {error}'
 
     def __init__(self, key, code, error_message, source_file=None):
-        self.key = key
+        self.key = self._get_display_key(key)
```

Comment:
The key shouldn't change. The modified key should only be used on the error msg

Reply:
Sure!